source/conf.py

```diff
@@ -58,12 +59,18 @@
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = 'en' 
+language = "en"
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path.
-exclude_patterns = ["_build", "modules.rst", "setup.rst"]
+exclude_patterns = [
+    "_build",
+    "modules.rst",
+    "setup.rst",
+    "prereise/demand/transportation_electrification/data.rst",
+    "prereise/demand/transportation_electrification/methodology.rst",
```

Comment:
We add this because the transportation branch hasn't been merged in PreREISE, right?

Reply:
No, it is because `:numref:` does not work in files included with `include` directive. The issue is solved there: https://github.com/sphinx-doc/sphinx/issues/9779

Reply:
I see.